### DISCUSSION TITLE
Multiple Objectives

### DISCUSSION THREAD

**Discussion by achillesrasquinha** [opencobra/cobrapy#1186] [General]
### Checklist

- [ ] There are [no similar issues or pull requests](https://github.com/opencobra/cobrapy/issues) for this yet.

## Describe the solution you would like.

I'm curious to know if there is a possible way of implementing a multiple-objective based solving?

**Comment by cdiener** [comment]
Yep, there is. For instance, [cameo](https://cameo.bio/) does a lot of it and it uses cobrapy. However, there is always the question of how to solve pareto-optimal solutions and often requires you to pick a strategy.
